7.8.вложенные_циклы_часть_1/7.8.9.py

```diff
@@ -0,0 +1,5 @@
+num = int(input())
+for i in range(1, num + 1):  
+    for j in range(i): 
```

Comment:
`j` не используешь, значит `_`:
```suggestion
    for _ in range(i): 
```